tests/tests_test_workflow/test_integ_workflow/integ_test/test_service_opensearch.py

```diff
@@ -62,7 +62,7 @@ def test_start(self, mock_tarfile_open: Mock, mock_dump: Mock, mock_file: Mock,
         # call test target function
         service.start()
 
-        mock_process.assert_called_once_with("export OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! && ./opensearch-tar-install.sh", os.path.join(self.work_dir, "opensearch-1.1.0"), False)
+        mock_process.assert_called_once_with("env OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123! && ./opensearch-tar-install.sh", os.path.join(self.work_dir, "opensearch-1.1.0"), False)
```

Comment:
need to remove `&&` here.

Reply:
Done in [5d8d699](https://github.com/opensearch-project/opensearch-build/pull/4334/commits/5d8d69961af26f42b24eb8a01a8870ac03b8aa9c)